cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/elements/NodeConversion.java

```diff
@@ -163,15 +164,13 @@ public void setVoltageAngleNodeBreaker() {
         VoltageLevel.NodeBreakerView topo = vl.getNodeBreakerView();
         String connectivityNode = id;
         int iidmNode = context.nodeMapping().iidmNodeForConnectivityNode(connectivityNode, vl);
-        if (!topo.hasAttachedEquipment(iidmNode)) {
-            LOG.error("ConnectivityNode {} with voltage and angle is not valid in IIDM", connectivityNode);
-            return;
-        }
```

Comment:
Why do you remove this log?

Reply:
@lcosson: any feedback?

Reply:
Very sorry about that, it looks like I miss-merged this file. Thank God for PR's!
Correcting...